tests/CommonTestUtils/AssertEx.cs

```diff
@@ -52,7 +42,7 @@ public static class AssertEx
         public static async Task<TActual?> ThrowsAsync<TActual>(AsyncTestDelegate code)
             where TActual : Exception
         {
-            return await ThrowsAsync<TActual>(code, string.Empty, null);
+            return await ThrowsAsync<TActual>(code, string.Empty, []);
```

Comment:
```suggestion
            return await ThrowsAsync<TActual>(code, string.Empty);
```
Empty `params` list should work, too.